backend/ml/management/commands/ml_train.py

```diff
@@ -160,14 +160,13 @@ def save_data(video_scores, contributor_rating_scores, poll, trusted_only=True):
         {(rating.user_id, rating.entity_id): rating.id for rating in created_ratings}
     )
 
+    query = ContributorRatingCriteriaScore.objects \
+        .filter(contributor_rating__poll_id=poll.pk)
     if trusted_only:
-        ContributorRatingCriteriaScore.objects.filter(
-            contributor_rating__poll_id=poll.pk
-        ).filter(contributor_rating__user__in=User.trusted_users()).delete()
+        query = query.filter(contributor_rating__user__in=User.trusted_users())
     else:
-        ContributorRatingCriteriaScore.objects.filter(
-            contributor_rating__poll_id=poll.pk
-        ).exclude(contributor_rating__user__in=User.trusted_users()).delete()
+        query = query.exclude(contributor_rating__user__in=User.trusted_users())
+    query.delete()
```

Comment:
Maybe a better name for `query` could be `ratings_to_delete` ?

Reply:
Yes, thanks ! :D Don't hesitate to "make a suggestion" about it next time, this will help me and will credit your as co author.